mysql-test/main/trigger.test

```diff
@@ -2237,15 +2237,15 @@ create trigger t1_bi before insert on t1 for each row begin end;
 create trigger t1_bi2 before insert on t1 for each row begin end;
 drop trigger t1_bi;
 drop trigger t1_bi2;
---error ER_TRG_DOES_NOT_EXIST
+--error 1360,1192
```

Comment:
Drop event was failing because there was a locked table. But it was inconsistent and I was getting both errors ER_TRG_DOES_NOT_EXIST or ER_LOCK_OR_ACTIVE_TRANSACTION. I fixed it by only running drop_event if there isn't an active locked table.